composer.json

```diff
@@ -20,7 +20,10 @@
         "nelmio/security-bundle": "^2.2",
         "contao/manager-plugin": "^2.1",
         "contao/core-bundle": "~4.4.0@dev",
-        "contao/installation-bundle": "~4.4.0@dev"
+        "contao/installation-bundle": "~4.4.0@dev",
+        "terminal42/header-replay-bundle": "^1.0",
+        "php-http/guzzle6-adapter": "^1.1",
+        "friendsofsymfony/http-cache": "^2.0"
```

Comment:
`friendsofsymfony/http-cache` is already required by `terminal42/header-replay-bundle`. Dunno why we should require `php-http/guzzle6-adapter` here either?